[Bug]: Build failure

### Summary

Unable to build on  linux (Ubuntu based)

### Affected component

tooling/ci

### Severity

S1 - workflow blocked

### Current behavior

```sh
rayan@dell:~/zeroclaw$ cargo build --release --locked
   Compiling zeroclaw v0.1.0 (/home/rayan/zeroclaw)
error[E0560]: struct `ChannelMessage` has no field named `reply_to`
   --> src/channels/lark.rs:453:25
    |
453 |                         reply_to: lark_msg.chat_id.clone(),
    |                         ^^^^^^^^ `ChannelMessage` does not have this field
    |
    = note: available fields are: `reply_target`

error[E0609]: no field `reply_to` on type `&ChannelMessage`
   --> src/gateway/mod.rs:712:57
    |
712 |                 if let Err(e) = wa.send(&response, &msg.reply_to).await {
    |                                                         ^^^^^^^^ unknown field
    |
    = note: available fields are: `id`, `sender`, `reply_target`, `content`, `channel`, `timestamp`

error[E0609]: no field `reply_to` on type `&ChannelMessage`
   --> src/gateway/mod.rs:721:30
    |
721 |                         &msg.reply_to,
    |                              ^^^^^^^^ unknown field
    |
    = note: available fields are: `id`, `sender`, `reply_target`, `content`, `channel`, `timestamp`

error[E0560]: struct `ChannelMessage` has no field named `reply_to`
  --> src/channels/cli.rs:43:17
   |
43 |                 reply_to: "user".to_string(),
   |                 ^^^^^^^^ `ChannelMessage` does not have this field
   |
   = note: available fields are: `reply_target`

error[E0560]: struct `ChannelMessage` has no field named `reply_to`
   --> src/channels/dingtalk.rs:241:25
    |
241 |                         reply_to: chat_id,
    |                         ^^^^^^^^ `ChannelMessage` does not have this field
    |
    = note: available fields are: `reply_target`

warning: unused variable: `clean_content`
   --> src/channels/discord.rs:358:25
    |
358 |                     let clean_content = if self.mention_only {
    |                         ^^^^^^^^^^^^^ help: if this is intentional, prefix it with an underscore: `_clean_content`
    |
    = note: `#[warn(unused_variables)]` (part of `#[warn(unused)]`) on by default

Some errors have detailed explanations: E0560, E0609.
For more information about an error, try `rustc --explain E0560`.
warning: `zeroclaw` (lib) generated 1 warning
error: could not compile `zeroclaw` (lib) due to 5 previous errors; 1 warning emitted
rayan@dell:~/zeroclaw$ 

```

### Expected behavior

Should complete the build succesfully

### Steps to reproduce

```bash
cargo build --release --locked
```

### Impact

Unable to build

### Logs / stack traces

```text

```

### ZeroClaw version

v0.1.0, 8ad5b6146ba3efc959bd5d7e9d09d3dd3159b96b

### Rust version

rustc 1.92.0 (ded5c06cf 2025-12-08)

### Operating system

Linux dell 6.8.0-60-generic #63~22.04.1-Ubuntu SMP PREEMPT_DYNAMIC Tue Apr 22 19:00:15 UTC 2 x86_64 x86_64 x86_64 GNU/Linux

### Regression?

Yes, it worked before

### Pre-flight checks

- [x] I reproduced this on the latest main branch or latest release.
- [x] I redacted secrets/tokens from logs.
- [x] I removed personal identifiers and replaced identity-specific data with neutral placeholders.

--- a/src/channels/cli.rs
+++ b/src/channels/cli.rs
@@ -40,7 +40,7 @@ impl Channel for CliChannel {
             let msg = ChannelMessage {
                 id: Uuid::new_v4().to_string(),
                 sender: "user".to_string(),
-                reply_to: "user".to_string(),
+                reply_target: "user".to_string(),
                 content: line,
                 channel: "cli".to_string(),
                 timestamp: std::time::SystemTime::now()

--- a/src/channels/dingtalk.rs
+++ b/src/channels/dingtalk.rs
@@ -238,7 +238,7 @@ impl Channel for DingTalkChannel {
                     let channel_msg = ChannelMessage {
                         id: Uuid::new_v4().to_string(),
                         sender: sender_id.to_string(),
-                        reply_to: chat_id,
+                        reply_target: chat_id,
                         content: content.to_string(),
                         channel: "dingtalk".to_string(),
                         timestamp: std::time::SystemTime::now()

--- a/src/channels/lark.rs
+++ b/src/channels/lark.rs
@@ -450,7 +450,7 @@ impl LarkChannel {
                     let channel_msg = ChannelMessage {
                         id: Uuid::new_v4().to_string(),
                         sender: lark_msg.chat_id.clone(),
-                        reply_to: lark_msg.chat_id.clone(),
+                        reply_target: lark_msg.chat_id.clone(),
                         content: text,
                         channel: "lark".to_string(),
                         timestamp: std::time::SystemTime::now()

--- a/src/gateway/mod.rs
+++ b/src/gateway/mod.rs
@@ -709,7 +709,7 @@ async fn handle_whatsapp_message(
         {
             Ok(response) => {
                 // Send reply via WhatsApp
-                if let Err(e) = wa.send(&response, &msg.reply_to).await {
+                if let Err(e) = wa.send(&response, &msg.reply_target).await {
                     tracing::error!("Failed to send WhatsApp reply: {e}");
                 }
             }
@@ -718,7 +718,7 @@ async fn handle_whatsapp_message(
                 let _ = wa
                     .send(
                         "Sorry, I couldn't process your message right now.",
-                        &msg.reply_to,
+                        &msg.reply_target,
                     )
                     .await;
             }

